test/controllers/agent/AgentHubControllerSpec.scala

```diff
@@ -179,10 +205,31 @@ class AgentHubControllerSpec extends ControllerBaseSpec
           Some(LocalDate.parse("2020-01-01")),
           isOverdue = false,
           payments = 2,
-          directDebitSetup = None
+          directDebitSetup = None,
+          shouldShowPenaltiesTile = false
+        )
+
+        val result = controller.constructViewModel(customerDetailsAllInfo, paymentsNotOverdue, None)(userWithBlueBox)
+        result shouldBe expected
+      }
+
+      "the user has penalties" in {
```

Comment:
Could you please also still add another test for when the user has no penalties?